src/iotjs_binding.cpp

```diff
@@ -116,13 +119,17 @@ JObject::~JObject() {
 
 
 JObject& JObject::Null() {
-  static JObject null(JVal::Null(), false);
-  return null;
+  if (null == NULL) {
+    null = new JObject(JVal::Null(), false);
```

Comment:
@akosthekiss I see.. Actually I found this when I tried to build nuttx, and then you occured to me there's possibility of linking problem with nuttx. I will check if there's other way to overcome this trouble then.